python/lsst/ts/lasertracker/laser_tracker_csc.py

```diff
@@ -474,23 +574,57 @@ async def measure_alignment(self, target: str) -> None:
 
         assert self.model is not None
 
-        await self.model.set_telescope_position(
-            telalt=self.elevation,
-            telaz=self.azimuth,
-            camrot=self.camrot,
-        )
+        await self.set_telescope_position()
 
         await self.model.measure_target("M1M3")
 
         if target != "M1M3":
             await self.model.measure_target(target)
 
+        target_frame_name = self.get_target_name(target)
+        reference_frame_name = self.get_target_name("M1M3")
+
         target_offset = await self.model.get_target_offset(
-            target=target, reference_pointgroup="M1M3"
+            target=target_frame_name, reference_pointgroup=reference_frame_name
         )
 
+        target_offset["force_output"] = True
         await self.evt_offsetsPublish.set_write(**target_offset)
 
+    async def set_telescope_position(self) -> None:
+        """Set the telescope positions by retrieving values from the mtmount
+        telemetry.
+        """
+        assert self.model is not None
+
+        self.elevation, self.azimuth, self.camrot = (
+            self.elevation_default,
+            self.azimuth_default,
+            self.camrot_default,
```

Comment:
camrot isn't set. Either create remote for camrot (MTRotator?) and set it, or probably drop assignment for camrot and assign it a default in constructor to clearly signal this isn't (yet?) handled.